.github/workflows/build-package.yml

```diff
@@ -3,84 +3,78 @@ name: Build Package
 on:
   workflow_call:
     inputs:
-      os:
-        description: The list of operating systems to use in the strategy matrix.
-        type: string
-        required: false
-        default: '["ubuntu18.04", "ubuntu20.04", "debian9"]'
-      variant:
-        description: The list of variants to use in the strategy matrix.
-        type: string
-        required: false
-        default: '[{"arch": "arm", "platform": "linux/arm/v7"}, {"arch": "arm64", "platform": "linux/arm64/v8"}, {"arch": "amd64", "platform": "linux/amd64"}]'
       artifact:
-        description: The artifact to publish.
+        description: The artifact output containing the packages from this workflow.
         type: string
         required: false
-        default: 'package'
+        default: "package"
       build-type:
-        description: The build type to use.
+        description: The type of build to perform (Debug | Release).
         type: string
-        default: Release
-        required: false
-      signed:
-        description: If the packages should be signed. This will create a second artifact for the signed packages (<artifact_name>-signed).
-        type: boolean
         required: false
-        default: true
+        default: "Release"
+      os:
+        description: The operating system to build for.
+        type: string
+        required: true
+      arch:
+        description: The architecture to build for.
+        type: string
+        required: true
+      platform:
+        description: The platform to build for.
+        type: string
+        required: true
+
+env:
+  BUILD_TYPE: Release
+  MOUNT: /azure-osconfig
+  REGISTRY: ghcr.io
```

Comment:
i suspect this will be an issue, even when running on a 1ES node, we can not guarantee container security through the `ghcr.io` we might want to find a public endpoint where we can use our secured images AND also allow forked builds to use these containers.